blog/electron-16-0.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Electron 16.0.0
+date: 2021-11-16T00:00:00.000Z
+authors:
+    - name: sofianguy
+      url: 'https://github.com/sofianguy'
+      image_url: 'https://github.com/sofianguy.png?size=96'
+    - name: ckerr
+      url: 'https://github.com/ckerr'
+      image_url: 'https://github.com/ckerr.png?size=96'
+slug: electron-16-0
+
+---
+
+Electron 16.0.0 has been released! It includes upgrades to Chromium `96`, V8 `9.6`, and Node.js `16.9.1`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 16.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+As of Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
+
+Additionally, Electron has changed supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://www.electronjs.org/docs/latest/tutorial/electron-versioning) for more detailed information about versioning in Electron. After May 2022, we will return to supporting latest three versions.
+
+### Stack Changes
+
+* Chromium `96`
+    * [New in Chrome 96](https://developer.chrome.com/blog/new-in-chrome-96/)
+* Node.js `16.9.1`
+    * [Node 16.9.1 blog post](https://nodejs.org/en/blog/release/v16.9.1/)
+* V8 `9.6`
+    * [V8 9.6 blog post](https://v8.dev/blog/v8-release-96)
+
+### Highlight Features
+
+* Now supports the [WebHID](https://developer.mozilla.org/en-US/docs/Web/API/WebHID_API) API. [#30213](https://github.com/electron/electron/pull/30213)
+* Add data parameter to `app.requestSingleInstanceLock` to share data between instances. [#30891](https://github.com/electron/electron/pull/30891)
+* Pass securityOrigin to media permissions request handler. [#31357](https://github.com/electron/electron/pull/31357)
+* Add `commandLine.removeSwitch`. [#30933](https://github.com/electron/electron/pull/30933)
+
+See the [16.0.0 release notes](https://github.com/electron/electron/releases/tag/v16.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+Below are breaking changes introduced in Electron 16. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/latest/breaking-changes) page.
+
+### Building Native Modules
+
+If your project uses node-gyp to build native modules, you may need to call it with `--force-process-config` depending on your project's setup and your Electron version. More information about this change can be found at [#2497](https://github.com/nodejs/node-gyp/pull/2497).
```

Comment:
Perhaps note that `electron-rebuild@^3.2.5` supports this automatically, so folks using `-rebuild` or Electron Forge only need to update their dependencies?